src-docs/spec/messenger.md

```diff
@@ -9,49 +9,68 @@ The same webxdc container file shared in two separate chat messages
 will be regarded as two separate "webxdc apps" 
 and the two apps can not communicate with, or even know about, each other. 
 
-### Webview Constraints for Running Apps
+### Webview Constraints
 
-When starting a web view for a webxdc app to run, messenger implementors:
+When starting a web view for a webxdc app to run, messenger implementors
 
-- MUST run the [webxdc container file](./format.md) in a constrained, 
-  network-isolated webview that 
-  MUST deny all forms of internet access. 
-  If you don't do this
-  unsuspecting users may leak data of their private interactions to outside third parties.
-  You do not need to offer "privacy" or "cookie" consent screens as
-  there is no way a webxdc app can implicitly transfer user data to the internet.
+- MUST serve all resources from the [webxdc container file](./format.md) 
+  to the webxdc application. 
+
```

Comment:
the information that `index.html` is the starting point is a bit implicit now. maybe amend sth. as:

```suggestion
- MUST serve all resources from the [webxdc container file](./format.md) 
  to the webxdc application. 

- on initial opening, MUST open root URL for the webview with `index.html` appended 
```